App/SB/views/Settings/index.js

```diff
@@ -63,60 +60,51 @@ class SyncPermissions extends React.PureComponent {
-      case 'alwaysAllow':
-        info = {
-          title: 'Choose, Always Allow',
-          details: 'In the location permission, please select, "Always Allow". It is needed by the app to periodically wake up and ensure you are getting updates to and from your peer network. Without it, the app will provide a lonely experience. We never collect, store, or share your location data.'
-        }
-        break
-      case 'camera':
-        info = {
-          title: 'Camera Roll',
-          details: 'Textile accesses your camera roll to import any new photos you take after you install the app. Without access to your camera roll, you will have no photos to view or share in the app. Photos added to Textile are privately encrypted - only visible to you ever - and hosted on IPFS. The only time they will ever be visible to anyone else is if you share them with your friends via our shared Threads feature.'
-        }
-        break
-      case 'background':
-      default:
-        info = {
-          title: 'Background location',
-          details: 'Background location allows Textile to wake up periodically to check for updates to your camera roll and to check for updates on your peer-to-peer network. Without background location the app will never get any new information, it will be a pretty boring place. We never keep, store, process, or share your location data with anyone or any device.'
-        }
-    }
+  showInfo (service: string) {
+    const info = GetServiceInfo(service)
     this.setState({infoVisible: true, info})
   }
 
   render () {
     return (
       <View style={styles.container}>
         <View style={styles.header}>
-          <Text style={styles.title}>Everything can run a bit better with a few permissions</Text>
+          <Text style={styles.title}>Choose the types of notifications you want to receive.</Text>
         </View>
         <ScrollView style={styles.contentContainer}>
           <View style={styles.listContainer}>
-            <View style={styles.listItem}>
-              <View>
-                <Text style={styles.itemTitle}>Background Location</Text>
-                <View style={styles.itemTexts}>
-                  <Text style={styles.itemDescription}>Wakes app up for updates</Text>
-                  <TouchableOpacity onPress={() => { this.showInfo('background') }}>
-                    <ImageSc width={15} source={require('./statics/icon-info.png')} />
-                  </TouchableOpacity>
-                </View>
-              </View>
-              <Button title={'enable'} color={'#4a4a4a'} buttonStyle={styles.permissionsButtonStyle} onPress={this.toggleBackground.bind(this)} />
-            </View>
+            {Object.keys(this.props.services)
+              .map((service, i) => {
+                const value = !!this.props.services[service].status
+                let children = Object.keys(this.props.children)
+                  .filter((key) => this.props.children[key].info.dependsOn === service)
```

Comment:
`dependsOn` is gone now, right?

Reply:
Oh I guess you removed it from the `ReduxPersist` migration, but it's still in `TextileTypes`. Not sure what your intention is there, let me know.

Reply:
nah, not in TTypes, i just append it here in this view. Adds an info object per key. Info contains what the title string, info string, nesting info are. All things just for the display. It's here, https://github.com/textileio/textile-mobile/pull/382/files#diff-ca34c21d7c542bf55f24da0b6f885ba9R1

Reply:
Ayyy, just ignore me for a minute, think i'm all confused about where different things are defined/used.